oryx-app-mllib/src/test/java/com/cloudera/oryx/app/mllib/kmeans/AbstractKMeansEvalTest.java

```diff
@@ -27,17 +27,16 @@
 import com.cloudera.oryx.app.kmeans.KMeansPMMLUtils;
 import com.cloudera.oryx.app.kmeans.KMeansPMMLUtilsTest;
 import com.cloudera.oryx.app.kmeans.SquaredDistanceFn;
-import com.cloudera.oryx.common.OryxTest;
 
-public abstract class AbstractKMeansEvalTest extends OryxTest {
```

Comment:
This is still changing the unit test. I really think they can be left alone. We are adding a new test class only.